pkg/sensors/manager.go

```diff
@@ -361,15 +362,17 @@ type tracingPolicyList struct {
 }
 
 type tracingPolicyDisable struct {
-	ctx     context.Context
-	name    string
-	retChan chan error
+	ctx       context.Context
+	name      string
+	namespace string
+	retChan   chan error
 }
 
 type tracingPolicyEnable struct {
-	ctx     context.Context
-	name    string
-	retChan chan error
+	ctx       context.Context
+	name      string
+	namespace string
```

Comment:
same as `tracingPolicyDelete`, please use a `collectionKey` type  here and update its callers.